cmd/internal/root.go

```diff
@@ -83,7 +85,7 @@ var rootCmd = &cobra.Command{
 			Namespace:          cfOperatorNamespace,
 			MetricsBindAddress: "0",
 			LeaderElection:     false,
-			Port:               int(port),
+			Port:               2999,
```

Comment:
default port is 2999, right? why need to hardcoded here?

Reply:
When installing using Helm, the port specified here [0] is taken as the value. It happens like this because helm installation creates a service which puts env variable `CF_OPERATOR_WEBHOOK_SERVICE_PORT` (2999) [0] and the viper takes this as port. This is wrong as we should give target port of the service to start the manager. so I hardcoded it for now. 

[0] https://github.com/cloudfoundry-incubator/cf-operator/blob/master/deploy/helm/cf-operator/templates/service.yaml#L9

Reply:
Can we put this in a constant so the name explains why it's hardcoded? `servicePort`, maybe? Hm, or maybe put a comment in the code to explain the difference between the two ports.

So one is for connecting to the k8s service, the other for the controller-runtime manager to open a listen socket for the webhook server?

Reply:
Done.